include/cloudbreak.bash

```diff
@@ -169,7 +169,7 @@ cloudbreak-delete-certs() {
 
 cloudbreak-conf-uaa() {
 
-    env-import UAA_DEFAULT_SECRET "cbsecret2015"
+    env-import UAA_DEFAULT_SECRET "$(date | checksum sha1)"
```

Comment:
On this way each cbd command has a different secret. What we need is one fix secret per cloudbreak deployment.